recipes/rejected/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set version = "3.12.2" %}
+
+package:
+  name: rejected
+  version: {{ version }}
+
+source:
+  fn: rejected-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/r/rejected/rejected-{{ version }}.tar.gz
+  md5: 1bd216f95124a4a29f3722fe00d1f800
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+  entry_points:
+    - rejected = rejected.controller:main
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - helper
+    - pika >=0.10.0
+    - psutil
+    - pyyaml
+    - tornado >=4.2,<4.3
+
+  run:
+    - python
+    - helper
+    - pika >=0.10.0
+    - psutil
+    - pyyaml
+    - tornado >=4.2,<4.3
+
+test:
+  imports:
+    - rejected
+  commands:
+    - rejected --help
+
+about:
+  home: https://rejected.readthedocs.io
+  license: BSD 3-clause
+  summary: an AMQP (RabbitMQ) consumer daemon and message processing framework
```

Comment:
I usually start with a capital letter here b/c it will read better in the README later on. (But that is a minor thing and this is good to go.)